src/app/chat/view.nim

```diff
@@ -348,7 +348,16 @@ QtObject:
         if msg.chatId != self.activeChannel.id:
           let channel = self.chats.getChannelById(msg.chatId)
           if not channel.muted:
-            self.messageNotificationPushed(msg.chatId, msg.text, msg.messageType, channel.chatType.int, msg.timestamp, msg.identicon, msg.alias, msg.hasMention)
+            self.messageNotificationPushed(
+              msg.chatId,
+              msg.text,
+              msg.messageType,
+              channel.chatType.int, 
+              msg.timestamp,
+              msg.identicon,
+              msg.alias, 
+              msg.hasMention,
+              channel.chatType.isOneToOne and self.status.contacts.isAdded(channel.id))
```

Comment:
Maybe you should rename the property `isContact` or only pass `self.status.contacts.isAdded(channel.id)`, since right now, the property name and the actual value don't match.

I think you could leave it as `isContact`and then do the check on the channel type in QML